apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -21,62 +22,93 @@ class AnimationPickerListItem extends React.Component {
   };
 
   state = {
-    loaded: false
+    loaded: false,
+    hover: false
   };
 
   render() {
-    const rootStyle = [styles.root, !this.props.label && styles.noLabel];
+    const {
+      icon,
+      animationProps,
+      category,
+      onClick,
+      playAnimations,
+      label
+    } = this.props;
+    const {loaded, hover} = this.state;
+    const rootStyle = [styles.root, !label && styles.noLabel];
 
     const thumbnailStyle = [
       styles.thumbnail,
-      this.props.icon && styles.thumbnailIcon,
-      this.props.animationProps && {
-        display: this.state.loaded ? 'block' : 'none'
+      icon && styles.thumbnailIcon,
+      animationProps && {
+        display: loaded ? 'block' : 'none'
       }
     ];
 
     const labelStyle = [
       styles.label,
-      this.props.icon && styles.labelIcon,
-      this.props.animationProps && {
-        display: this.state.loaded ? 'block' : 'none'
+      icon && styles.labelIcon,
+      animationProps && {
+        display: loaded ? 'block' : 'none'
       }
     ];
-    const iconImageSrc = this.props.category
-      ? `/blockly/media/p5lab/animation-previews/category_${
-          this.props.category
-        }.png`
+    const iconImageSrc = category
+      ? `/blockly/media/p5lab/animation-previews/category_${category}.png`
       : '';
 
+    const centerStyle = {
+      top: THUMBNAIL_SIZE / 2 - 12,
+      left: THUMBNAIL_SIZE / 2 - 12
+    };
+
+    const hoverIcon = [styles.hoverIcon, centerStyle];
+
+    const hoverBorder = {
+      borderStyle: 'solid',
+      borderRadius: 12,
+      cursor: 'pointer',
+      borderColor: color.purple,
+      borderWidth: '3px',
+      padding: 0
+    };
+
+    const displayMultiSelect = experiments.isEnabled(experiments.MULTISELECT);
+
+    const thumbnailStyleWithHover = [thumbnailStyle, hover && hoverBorder];
+
     return (
       <div
         style={rootStyle}
-        onClick={this.props.onClick}
+        onClick={onClick}
         className="uitest-animation-picker-item"
+        onMouseEnter={() => this.setState({hover: true})}
+        onMouseLeave={() => this.setState({hover: false})}
       >
-        <div style={thumbnailStyle}>
-          {this.props.animationProps && (
+        <div style={thumbnailStyleWithHover}>
```

Comment:
i know this is on a deadline, so this can be follow-up work, but i think we should abstract this hover behavior into its own component, and i don't think it would require a ton of refactoring. 

brainstorming some thoughts on how this could work below. this solution won't work directly, but i'm showing you how i'm thinking about it:

```javascript
// sorry for the name, my brain is having a hard time naming right now
function HoverComponent(props) {
  return (
    <div style={[props.style, styles.default, props.hover && styles.hover]}>
      {props.children}
      // icon probably needs to live in a separate/sibling element?
      {withIcon && <i className="fa fa-plus fa-2x" style={styles.icon} />}
    </div>
  );
}

const styles = {
  default: {
    // border styles for when we aren't hovering
  },
  hover: {
    // border styles on hover (currently defined by hoverBorder in AnimationPickerListItem)
  },
  icon: {
    // icon styles (current defined by styles.hoverIcon in AnimationPickerListItem)
  }
};
```

and in `<AnimationPickerListItem/>` the usage of this component could look something like:

```javascript
<HoverComponent
  style={thumbnailStyle}
  hover={hover}
  withIcon={animationProps && loaded && hover && displayMultiSelect}
>
  {animationProps && <AnimationPreview />}
  {icon && <i className={'fa fa-' + icon} />}
  {category && <img />}
</HoverComponent>
```